cmd/airflow.go

```diff
@@ -91,6 +92,7 @@ func init() {
 
 	// Airflow deploy
 	airflowRootCmd.AddCommand(airflowDeployCmd)
+	airflowDeployCmd.Flags().BoolVarP(&forceDeploy, "force-deploy", "f", false, "Force deploy if uncommited changes")
```

Comment:
@andscoop what do you think about just using `force` for the long option name, instead of `force-deploy`? Feels more standard.